pipeline.Dockerfile

```diff
@@ -24,6 +25,7 @@ RUN --mount=type=cache,target=/root/.cache/pip \
 
 # Add repository code
 COPY pipeline/ /opt/dagster/app/pipeline/
+COPY scripts/ /opt/dagster/app/scripts
```

Comment:
Did you omit the trailing `/` on purpose?
(I always forget the different `COPY`/`ADD` notations...)

Reply:
Added